test/MUIDataTableBody.test.js

```diff
@@ -493,4 +533,69 @@ describe('<TableBody />', function() {
 
     expect(html).to.contain('Test_Text');
   });
+
+  it('should isRowSelectable be used to determine if row is selectable', () => {
```

Comment:
Actually, I think we can eliminate this test, or recombine it. (or change it as I'll discuss below). It looks like it has a couple of issues once separated out. Mainly, it just seems to be testing whether or not the `isRowSelectable` function we create here in the test has the proper return values, which basically means that the test is testing itself. It doesn't test that the table works because it doesn't simulate an interaction with the table select rows.

So, if we just want to show that the function is called, the test below (on line 571) does that, so we could combine it back with that one.

If instead we want to show the positive case, where we demonstrate that a click on a selected row that is allowed by `isRowSelectable` works, then I think we should model it after the one on line 274 and simulate the clicks.

I leave it to you as for how you'd like to resolve this one.